src/drivers/ocpoc_sbus_rc_in/CMakeLists.txt

```diff
@@ -0,0 +1,43 @@
+############################################################################
```

Comment:
As a general question for the additions here: Would this driver be reusable as a general driver? If so we should remove the ocpos_ prefix.